app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/ui/tracker_activity/DeviceShieldActivityFeedFragment.kt

```diff
@@ -207,7 +207,7 @@ class DeviceShieldActivityFeedFragment : DuckDuckGoFragment() {
     companion object {
         private val defaultConfig = ActivityFeedConfig(
             maxRows = Int.MAX_VALUE,
-            timeWindow = 5,
+            timeWindow = 7,
```

Comment:
Changed from 5 to 7 as agreed in comments on https://app.asana.com/0/0/1204022302262178/f